feature-set-interface/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+//! Collection of all runtime features.
```

Comment:
This doc comment isn't relevant to this interface crate

Reply:
removed the whole block